utils/post.js

```diff
@@ -353,7 +366,7 @@ async function filterFeeds(
       let newItem = {...item};
 
       newItem.is_self = item?.actor?.id === userId || item?.actor?.id === selfAnonymousUserId;
-
+      newItem = await modifyFeedIsFollowingTarget(newItem, userId);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This line calls the `modifyFeedIsFollowingTarget` function which is an asynchronous operation. However, it's not being awaited here, which might lead to unexpected behavior because the promise returned by this function may not be resolved when the next lines of code execute. Make sure to use `await` keyword when calling this function.
```diff
- newItem = modifyFeedIsFollowingTarget(newItem, userId);
+ newItem = await modifyFeedIsFollowingTarget(newItem, userId);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->